skiros2/owl/skiros.owl

```diff
@@ -852,6 +852,13 @@
     </owl:Class>
 
 
+    <!-- http://rvmi.aau.dk/ontologies/skiros.owl#AvailableForPlanning -->
+
+    <owl:DatatypeProperty rdf:about="http://rvmi.aau.dk/ontologies/skiros.owl#Skill">
+        <rdfs:domain rdf:resource="http://rvmi.aau.dk/ontologies/skiros.owl#Skill"/>
+        <rdfs:range rdf:resource="http://www.w3.org/2001/XMLSchema#boolean"/>
+    </owl:DatatypeProperty>
```

Comment:
this is just a comment that is always there for some reason. Maybe a tool like protege makes them.
The actual definition is below, so this defined a `skiros:Skill` property and not `skiros:AvailableForPlanning`

SkiROS2 does not check strongly for types and that's why it didn't blow up

Reply:
good eye!